terraform/aws/modules/network/main.tf

```diff
@@ -108,14 +108,14 @@ resource "aws_security_group" "km_alb_sg" {
     protocol    = "tcp"
     from_port   = 443
     to_port     = 443
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = ["<cidr>"]
   }
 
   ingress {
     protocol    = "tcp"
     from_port   = 80
     to_port     = 80
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = ["<cidr>"]
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure Security Groups do not have unrestricted specific ports open - (HTTP,80)**
 
**Vulnerability**
Security Groups-Unrestricted Specific Ports -HTTP,80.
 
**Remediation**
Change cidr_block to non 0.0.0.0/0.

source: accurics-